addon/components/ember-tr/template.hbs

```diff
@@ -1,21 +1,39 @@
-{{#each cells as |cell|}}
+{{#each cells as |api|}}
   {{#if hasBlock}}
-    {{#if api.isHeader}}
+    {{#if isHeader}}
       {{yield (hash
-        api=cell
+        columnValue=api.columnValue
+        columnMeta=api.columnMeta
 
-        cell=(component "ember-th" api=cell)
+        enableResize=api.enableResize
```

Comment:
some options here do not seem documented

Reply:
Yes, that was intentional. Parts of the API are public, and parts aren't really and are just for communication. In the future ideally we will figure out a way to make these more genuinely private.